index.js

```diff
@@ -3,12 +3,13 @@
 const path = require('path')
 const fs = require('fs')
 const readPackage = require('read-package-json')
-const builtins = require('builtins')()
+const builtins = require('module').builtinModules
```

Comment:
It would be nicer to keep using `builtins` for supporting passing a node version, but NBD.

Reply:
True, but that can now be achieved by simply running it on another version, using eg. a Travis CI build matrix